dashboard/app/views/levels/_external.html.haml

```diff
@@ -2,6 +2,12 @@
 - last_attempt = @last_attempt unless local_assigns.has_key? :last_attempt
 - in_level_group ||= false
 - level_props = level.properties_with_replaced_markdown(current_user)
+- video_key = @level.video_key
+- video =  video_key ? Video.current_locale.find_by_key(video_key) : nil
+- use_large_video_player = level.properties["use_large_video_player"]
+- page_width = 800
```

Comment:
This constant is a bit confusing because it only applies when showing small video. I'd suggest moving it to under `- if video` below, or else renaming it to `small_video_page_width`.